src/App.css

```diff
@@ -7,6 +7,5 @@
 }
```

Comment:
- [ ] It would be nice if make a new pull request because your project will get a fresh review from the start so it would be nice if you make a fresh pull request from the styling branch and submit that new pull request for a review 👍